pmd-ant/src/main/java/net/sourceforge/pmd/ant/Formatter.java

```diff
@@ -44,6 +44,7 @@ public class Formatter {
     private final List<Parameter> parameters = new ArrayList<>();
     private Writer writer;
     private Renderer renderer;
+    private static final String ENCODING = "encoding";
```

Comment:
Extract the "encoding" string as a field variable can increase the maintainability. LGTM.  The Build Failure comes from the PMD's master branch, not something introduced by the PR.